tests/network/conftest.py

```diff
@@ -306,19 +308,22 @@ def _verify_jumbo_frame():
     def _verify_sriov():
         if any(test.get_closest_marker("sriov") for test in collected_tests):
             LOGGER.info("Verifying if the cluster supports running SRIOV tests...")
-            if not Namespace(name=py_config["sriov_namespace"], client=admin_client).exists:
+            if not sriov_namespace.exists:
                 failure_msgs.append(
-                    f"SRIOV operator is not installed, the '{py_config['sriov_namespace']}' namespace does not exist"
+                    f"SRIOV operator is not installed, the '{sriov_namespace.name}' namespace does not exist"
```

Comment:
If the namespace doesn't exist, then its `.name` member won't exist as well.

Reply:
Right, the`Namespace` object is created with a `name` member, regardless if represents an existing k8s resource. I see that know.
Please resolve.